test/functional/wallet_bumpfee.py

```diff
@@ -92,6 +93,38 @@ def run_test(self):
         test_small_output_with_feerate_succeeds(self, rbf_node, dest_address)
         test_no_more_inputs_fails(self, rbf_node, dest_address)
 
+def test_invalid_parameters(node, dest_address):
+    txid = spend_one_input(node, dest_address)
+    # invalid estimate mode
+    assert_raises_rpc_error(-8, "Invalid estimate_mode parameter", node.bumpfee, txid, {
+        "estimate_mode": "moo",
+    })
+    assert_raises_rpc_error(-3, "Expected type string", node.bumpfee, txid, {
+        "estimate_mode": 38,
+    })
+    assert_raises_rpc_error(-3, "Expected type string", node.bumpfee, txid, {
+        "estimate_mode": {
+            "foo": "bar",
+        },
+    })
+    assert_raises_rpc_error(-8, "Invalid estimate_mode parameter", node.bumpfee, txid, {
+        "estimate_mode": Decimal("3.141592"),
+    })
+    # confTarget and conf_target
+    assert_raises_rpc_error(-8, "confTarget and conf_target options should not both be set", node.bumpfee, txid, {
+        "confTarget": 123,
+        "conf_target": 456,
+    })
+    # confTarget and totalFee (depends on -deprecatedrpc=totalFee)
+    assert_raises_rpc_error(-3, "Unexpected key totalFee", node.bumpfee, txid, {
+        "confTarget": 123,
+        "totalFee": 456,
+    })
+    # conf_target and totalFee (depends on -deprecatedrpc=totalFee)
+    assert_raises_rpc_error(-3, "Unexpected key totalFee", node.bumpfee, txid, {
+        "conf_target": 123,
+        "totalFee": 456,
+    })
```

Comment:
69823cc can omit lines 118-127 as the `totalFee` argument is now removed

Reply:
Thanks, done!